custom_components/steam_wishlist/util.py

```diff
@@ -30,15 +31,49 @@ def get_steam_game(game_id: int, game: Dict[str, Any]) -> SteamGame:
         # Price is an integer so $6.00 is 600.
         sale_price = round(int(pricing["price"]) * 0.01, 2)
 
+    reviews_percent = game.get('reviews_percent', 'N/A')
+    review_desc = game.get('review_desc', 'No reviews')
+    rating_info = f"Reviews:&nbsp;&nbsp;{reviews_percent}% ({review_desc})"
+
+    tags = game.get("tags", [])
+    tags_string = ", ".join(tags)
+
+    try:
+        original_price = float(normal_price if normal_price is not None else 0)
+        sale_price_val = float(sale_price if sale_price is not None else original_price)
+        discount_percentage = int(discount_pct) if discount_pct is not None else 0
+        if original_price == 0:
+            price_info = "Price:&nbsp;&nbsp;TBD"
+        else:
+            original_price_formatted = f"{original_price:.2f}"
+            if sale_price is not None:
+                strikethrough_price = ''.join(ch + "\u0336" for ch in original_price_formatted[:-1]) + original_price_formatted[-1]
+                price_info = f"{strikethrough_price} ${sale_price_val:.2f} ({discount_percentage}% off)&nbsp;&nbsp;🎫"
+            else:
+                price_info = f"Price:&nbsp;&nbsp;${original_price:.2f}"
+    except (ValueError, TypeError):
+        price_info = "Price information unavailable"
+
+
+    release_date = ("Release date:&nbsp;&nbsp;" + datetime.datetime.utcfromtimestamp(int(game.get("release_date", "0"))).strftime("%b %d, %Y") + "&nbsp;&nbsp;🆕" if datetime.datetime.utcnow() < datetime.datetime.utcfromtimestamp(int(game.get("release_date", "0")) + 86400) else "Released:&nbsp;&nbsp;" + datetime.datetime.utcfromtimestamp(int(game.get("release_date", "0"))).strftime("%b %d, %Y")) if str(game.get("release_date", "0")).isdigit() else "Unknown"
```

Comment:
This is really hard to read, I'd factor this out into a new function. Generally speaking I wouldn't do more than a single ternary in a statement.

Also note that `datetime.utcfromtimestamp` and `datetime.utcnow` are deprecated.  Pass in the timezone instead.  e.g.

```python

from datetime import datetime, timezone

# replace datetime.utcfromtimestamp() with:
datetime.fromtimestamp(timestamp, timezone.utc)

# replace datetime.utcnow() with:
datetime.now(timezone.utc)
```

See: https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated
```

Reply:
Done.